documentation/migration-guides/Stack/migration-guide.1.2.12.md

```diff
@@ -0,0 +1,113 @@
+# Table of Contents
```

Comment:
Do we need a list of known issues?

Reply:
known issues will be in the main documentation site. I think it makes sense to add it here as well. We can do that later. I still expect us to edit this for few details even after the release(like adding specific examples Old vs New )